py/frequenz/client/weather/_types.py

```diff
@@ -25,6 +25,11 @@ class ForecastFeature(enum.Enum):
     UNSPECIFIED = weather_pb2.ForecastFeature.FORECAST_FEATURE_UNSPECIFIED
     """Unspecified forecast feature."""
 
+    TEMPERATURE_2_METRE = (
+        weather_pb2.ForecastFeature.FORECAST_FEATURE_TEMPERATURE_2_METRE
+    )
+    """Temperature at 2m altitude."""
```

Comment:
This is 2m above earth's surface, altitude is usually above sea level. Introduced that myself in the proto specs, fixed here: https://github.com/frequenz-floss/frequenz-api-weather/pull/75

Reply:
done